@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -20,6 +23,42 @@ function overlayObject(object: TileObject, displacementMap: THREE.DataTexture):
     }
 }
 
+function overlayPosition(
+    tileCoords: THREE.Vector3,
+    elevationProvider: ElevationProvider,
+    projection: Projection,
+    tile: Tile
+) {
+    // TODO: Calculate geoCoordinates in decoder.
+    const worldCoords = tileCoords.clone().add(tile.center);
```

Comment:
We should reduce creation of temporary objects. Please create a follow up ticket, if you did not already, and add that.

Reply:
i'll fix it here.